### PR TITLE
Keyboardtype modifier

### PR DESCRIPTION
**Pull request by Digital-Brendan**
### Description

I wrote a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier .keyboardType() in Swift, giving a simple example and outlining the correct usage & syntax.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md:16]:
```suggestion
The **`.keyboardType()`** view modifier method in SwiftUI is used to specify the keyboard type from the **UIKeyboardType** enumeration to use for text entry.
```

File: content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md:3]:
```suggestion
Description: 'Specifies the keyboard type to use for text entry.'
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Digital-Brendan I only see a couple of minor changes for this entry.

**Comment by Digital-Brendan** [comment]
No problem! I'll add the changes right away.

**Review by KTom101 — APPROVED** [review]
@Digital-Brendan This looks good for a second review 👍 Please wait for another maintainer to take a look at this.

File: content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md:29]:
```suggestion
- `type`: One of the keyboard types defined in the `UIKeyboardType` enumeration.
```

File: content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/keyboardType/keyboardType.md:33]:
```suggestion
The following example demonstrates how to use the `.keyboardType()` method to ensure the user can enter a correctly formatted email address:
```

**Comment by Digital-Brendan** [comment]
Thanks for pointing those out @SSwiniarski ! They should be corrected now.

**Review by SSwiniarski — APPROVED** [review]
@Digital-Brendan, looks good for another review.

**Comment by SSwiniarski** [comment]
@Digital-Brendan, congrats on the new entry. The live entry can be found here: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/keyboardType

image

**Comment by Digital-Brendan** [comment]
@SSwiniarski Sorry for digging this back up but I noticed I associated this with the wrong Codecademy account, meaning the contributor listed on the docs page isn't me! My codecademy account is actually `@Digital-Brendan`. Thanks

**Comment by SSwiniarski** [comment]
@KTom101, @yangc95, @caupolicandiaz Any idea of what to do here?

**Comment by KTom101** [comment]
@Digital-Brendan Can you remove the GitHub username under the brendan5024346943 Codecademy account within the profile details and then relink your GitHub username to the correct account using the steps below:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

Also, please make sure that you have the email associated with the correct Codecademy account listed as one of your emails under your GitHub profile.

**Pull request merged**
